python/src/aiconfig/editor/client/src/components/prompt/PromptMenuButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Menu, createStyles } from "@mantine/core";
+import { IconDotsVertical, IconTrash } from "@tabler/icons-react";
+import { memo } from "react";
+
+const useStyles = createStyles((theme) => ({
+  cellAction: {
+    marginLeft: -8,
+    [theme.fn.smallerThan("sm")]: {
+      float: "left",
+      marginTop: -2,
+      marginLeft: 0,
+      zIndex: 99,
+      "&:hover": {
+        backgroundColor:
+          theme.colorScheme === "dark"
+            ? "rgba(0, 0, 0, 0)"
+            : "rgba(255, 255, 255, 0)",
+      },
+    },
+  },
+}));
+
+export default memo(function PromptMenuButton({
+  promptId,
+  onDeletePrompt,
+}: {
+  promptId: string;
+  onDeletePrompt: (id: string) => void;
+}) {
+  const { classes } = useStyles();
+
+  return (
+    <Menu position="bottom-end">
```

Comment:
Nit: could we wrap this inside of a more visible border/box container? It feels now a bit like the `Delete Prompt` dropdown item is just sort of floating below without firmly being connected to the three vertical dots icon
<img width="218" alt="Screenshot 2023-12-29 at 10 37 55" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/67b76663-fb84-4a0f-84de-59684ed40b0d">

Reply:
Uh, not sure off the top of my head how to do that. But since this placement/UI isn't final anyway I'll hold off on anything for this